docs/README.md

```diff
@@ -66,6 +66,8 @@ If you are already familiar with SkyWalking, you could use this catalog to find
       * [Plugin development guide](en/setup/service-agent/java-agent/README.md#plugin-development-guide).
       * [Agent plugin tests and performance tests](en/setup/service-agent/java-agent/README.md#test).
     * [Other language agents](en/setup/README.md#language-agents-in-service) includes Nginx LUA, Python, .NetCore, PHP, NodeJS, Go.
+    * Browser performance monitoring
+      * Track the performance of the browser, such as latency of redirect, dns, ttfb. For more information, [click here](https://github.com/apache/skywalking-client-js).
```

Comment:
@arugal I fixed this doc. The browser agent has nothing related to distributed tracing.

@kezhenxu94 @ascrutae I hope either/both of you could check, whether your nodejs agents could work in the browser?